src/app/person/person.cpp

```diff
@@ -52,6 +52,16 @@ bool Person::validatePesel(const std::string& pesel) const
     return true;
 }
 
+bool Person::validatePesel() const
+{
+    if (not validatePeselSize(pesel_) or not validatePeselDate(pesel_) or not validateControlNumber(pesel_) or
+        not validateGender())
+    {
+        return false;
+    }
+    return true;
+}
+
```

Comment:
validatePesel(const std::string& pesel) 
is the same but without validateGender, please keep one definition of validation